darwin/dataset/remote_dataset.py

```diff
@@ -100,7 +101,10 @@ def pull(self, blocking: Optional[bool] = True, multi_threaded: Optional[bool] =
                     print(f"Could not remove dataset in {annotations_dir}. Permission denied.")
             annotations_dir.mkdir(parents=True, exist_ok=False)
             z.extractall(annotations_dir)
-            annotation_format = "json"
+            # Rename all json files pre-pending 'V7_' in front of them.
+            # This is necessary to avoid overriding them in a later stage.
+            for annotation_path in annotations_dir.glob(f"*.{annotation_format}"):
+                annotation_path.rename(annotation_path.parent / f"V7_{annotation_path.name}")
```

Comment:
Whatever prepended string you choose, we should probably get rid of it after all the annotations have been renamed.

Reply:
It is the case :)